config/sync/migrate_plus.migration.group_media_video.yml

```diff
@@ -19,13 +19,20 @@ source:
 process:
   type:
     plugin: default_value
-    default_value: group_content_type_7d15138adb66b
+    default_value: group_content_type_770174f09a085
   gid:
     -
       plugin: default_value
       default_value: 1
-  label: title
-  entity_id: fid
+  label: filename
+  entity_id:
```

Comment:
If the d7_file_media_video migration's VideoUrl.php migrate process plugin fails to fetch a thumbnail image, it used to ignore the item and skip on. This causes a problem because this migration uses the D7 source as the guide, then tries to create group_content relation entities... and if we've skipped a video entity we can't then load it in D9 to create the relation... you get a big nasty group or contententitybase error. That's quite obscure to debug, too.